### ISSUE TITLE
Cannot scan App Storage

### ISSUE THREAD

**Issue by Pictor13** [IvanVolosyuk/diskusage#34]
Hi,

the app works nice on every target except for the AppStorage.
While scanning it will always stuck (forever) on some app. I tried to uninstall all of those causing the issue but now is getting stuck to some SmartcardReader app that I guess is a phone component.

How can it that be solved? 
I have a hauwei y300.

Thanks!

**Comment by IvanVolosyuk** [comment]
Yeah, I have several reports for Hauwei devices. I have to figure out how
to deal with it

**Comment by IvanVolosyuk** [comment]
Newer versions of DiskUsage should report correctly the app which DiskUsage stuck at.

**Comment by Pictor13** [comment]
Sorry, I cannot test it!
I've lost my Huawei just one week after opening this issue 😅

But thank you for your work; I'll try it on my new Lenovo 🙃
